sky/execution.py

```diff
@@ -1025,7 +1025,9 @@ def serve_up(
         controller_task.best_resources = controller_best_resources
 
         controller_envs = {
+            'SKYPILOT_USER_ID': common_utils.get_user_hash(),
```

Comment:
@Michaelvll I'm not super familiar with this. could you comment on what are the envs to pass?

Reply:
This is the same implementation with `spot` in `spot-controller.yml.j2`

Reply:
got it, I think it should be good